### ISSUE TITLE
CSS not showing up?

### ISSUE THREAD

**Issue by kristasmith88** [Media-Ed-Online/intro-web-dev#99]
I have been working on this project all week and it has been showing up when I right click to  "open in browser" but I check my links after submitting in hw and its not working? I double checked my code and cant figure out what I am missing...

<img width="1392" alt="screen shot 2017-11-06 at 6 33 14 pm" src="https://user-images.githubusercontent.com/31778785/32469613-7577ddb2-c321-11e7-9586-8bfb203bb66a.png">
<img width="655" alt="screen shot 2017-11-06 at 6 32 54 pm" src="https://user-images.githubusercontent.com/31778785/32469616-76eedd30-c321-11e7-9582-9b74c6c336b8.png">

**Comment by MarkLannenUM** [comment]
I can't get my updated site to publish either. There is an old version (that looks terrible) that is live now, but the publish I did this afternoon just won't show up. 

And now Github desktop does not seem to recognize any changes that I have made.

If anyone has suggestions I'd really like to hear them!

Thanks

**Comment by JustineEvansUM** [comment]
Just wait a bit. GitHub, like most sites, can get bogged down and take minutes to hours to update models. On GitHub, check your repo to see if your changes are listed in the history - this means you successfully pushed, and GH might just be behind.

**Comment by MarkLannenUM** [comment]
Hi Justine

Something really wrong has happened with my project. I pushed it on Github desktop, and not only is an older version showing up on github, but the actual files on my computer have reverted to a what I had finished by yesterday and all of my work that I did today seems lost. Do you have any idea what could have happened or if any of this is salvageable?

My contact page is completely blank and a bunch of styling that I did today has disappeared...

**Comment by MarkLannenUM** [comment]
I don't think it is that. The actual files on my hard drive have been changed by github desktop (as far as I can tell) and all the changes I made today (and been saving) have been wiped and are back to what they looked like yesterday, but with something that looks like random markdown added to them.


Any ideas?

________________________________

**Comment by mjohnson92** [comment]
If anyone is still struggling with this... please.. check your CAPITALS like I did. :)

href="../css/style.css"> is different from href="../CSS/style.css">

**Comment by JohnYingling** [comment]
If it makes you feel better, I also have no idea what is going on with any of this.

**Comment by FriedmanM** [comment]
I had an error when pushing my project to GitHub last Sunday (unspecified error). If your stuff is missing you can check your Commits and see if you can revert back to where it was when you committed with the correct page layout.

**Comment by JustineEvansUM** [comment]
Hey team! @FriedmanM, @JohnYingling @MarkLannenUM @kristasmith88 

Were you all able to reach any conclusions here? Isn't the internet wonderful sometimes!

**Comment by kristasmith88** [comment]
Hi Justine!

Mine worked but like you said, I had to wait a little while before it would show up last week. I am still having issues with my font showing up but am sure it is because of my coding, after the CSS style started working. Im still trying to figure it out but am hoping its just going to take a little while before it updates. Thank you for checking in!!




Krista K Smith

________________________________

**Comment by JustineEvansUM** [comment]
Hey, Krista! Just keep me updated. If things still aren't working, I'll take a look!
